codecov.yml

```diff
@@ -1,16 +0,0 @@
-# https://docs.codecov.io/docs/codecov-yaml
```

Comment:
We don't use the CodeCov mechanism currently. There are plans to add a more standard mechanism in. See: https://github.com/dotnet/project-system/issues/8219